src/services/PriceSignerService.ts

```diff
@@ -0,0 +1,22 @@
+import {injectable, inject} from 'inversify';
+import {ethers} from 'ethers';
+
+import {DeviationSignerRepository} from '../repositories/DeviationSignerRepository.js';
+import {ChainsIds} from '../types/ChainsIds.js';
+
+@injectable()
+class PriceSignerService {
+  @inject(DeviationSignerRepository) protected deviationSignerRepository!: DeviationSignerRepository;
+
+  async sign(message: string): Promise<{signerAddress: string; hashVersion: number; signature: string; hash: string}> {
+    const hashVersion = 1;
+    const hash = ethers.utils.id(message);
+    const signer = this.deviationSignerRepository.get(ChainsIds.ROOTSTOCK);
```

Comment:
```suggestion
    const signer = this.deviationSignerRepository.get(ChainsIds.ETH);
```
lets use ETH, this is our main blockchain.